contracts/src/helpers/PBHExternalNullifier.sol

```diff
@@ -52,7 +65,8 @@ library PBHExternalNullifier {
     /// @dev This function ensures the external nullifier matches the current year and month,
     ///      and that the nonce does not exceed `numPbhPerMonth`.
     function verify(uint256 externalNullifier, uint8 numPbhPerMonth) public view {
```

Comment:
Found an issue here! We're actually under constrained. We need to make sure that the leading zeros here are as expected.
```suggestion
    function verify(uint256 externalNullifier, uint8 numPbhPerMonth) public view {
     require(externalNullifier <= type(uint40).max, InvalidExternalNullifierLeadingZeros());
```suggestion

Reply:
Thanks!